github/config.go

```diff
@@ -192,7 +192,7 @@ func getPassword() (string, error) {
 	}
 
 	c := make(chan os.Signal)
-	signal.Notify(c, os.Interrupt, os.Kill)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
```

Comment:
Basically, we want to restore terminal state if <kbd>Ctrl-C</kbd> or <kbd>Ctrl-D</kbd> are used. Since <kbd>Ctrl-C</kbd> is already handled as `os.Interrupt`, I guess it's okay to handle the additional TERM signal (you're right, Kill can't be handled).